husky_components/column-navigation/main.js

```diff
@@ -1281,13 +1299,46 @@ define(function() {
         },
 
         /**
+         * Returns all the marked objects.
+         */
+        mark: function(id) {
+            var $element = this.$find('li[data-id="' + id + '"]');
+            if (!!$element.length) {
+                // if single markable unmark all the others
+                if (this.options.singleMarkable === true) {
+                    $.each(this.marked, function (key, value) {
+                        this.unmark(value);
+                    }.bind(this));
+                }
+                this.sandbox.dom.addClass($element, constants.markedClass);
+                this.marked.push(id);
+                this.setItemsTextWidth($element);
+                // change unmark to mark icon
+                var $markButton = $element.find('span:last-child > .'+ this.options.actionIcon);
+                $markButton.removeClass(this.options.actionIcon);
+
+                /* prepend icon class */
+                var classes = $markButton.attr('class');
+                classes = this.options.unmarkIcon + ' ' + classes;
+                $markButton.attr('class', classes);
```

Comment:
It added the class but the icon wouldn´t show. But when I copied the class as the first class it worked.